src/topology/continuous_function/compact.lean

```diff
@@ -68,6 +69,36 @@ metric_space.induced
   (equiv_bounded_of_compact α β).injective
   (by apply_instance)
 
+section
+variables {α β} (f g : C(α, β)) {C : ℝ}
+
+/-- The distance between two functions is controlled by the supremum of the pointwise distances -/
+lemma dist_le (C0 : (0 : ℝ) ≤ C) : dist f g ≤ C ↔ ∀x:α, dist (f x) (g x) ≤ C :=
+@bounded_continuous_function.dist_le  _ _ _ _
+  ((equiv_bounded_of_compact α β) f) ((equiv_bounded_of_compact α β) g) _ C0
+
+lemma dist_le_of_nonempty [nonempty α] :
+  dist f g ≤ C ↔ ∀ x, dist (f x) (g x) ≤ C :=
```

Comment:
This name seems odd - the statement is essentially the same as `dist_lt_iff_of_nonempty`, but one has `iff` in the name and the other does not.

Reply:
I had just been following the names used in `bounded_continuous_function`. I've added an extra `_iff` in both places.

This does leave a discrepancy still: perhaps `dist_le` itself should be renamed to `dist_le_iff`? (In both `bounded_continuous_function` and `continuous_map`.)